geopyspark-backend/geotrellis/src/main/scala/geopyspark/geotrellis/util/GeometryUtils.scala

```diff
@@ -0,0 +1,10 @@
+package geopyspark.geotrellis.util
+
+import geotrellis.vector.Geometry
+import geotrellis.vector.io.wkb.WKB
+
+object GeometryUtil {
+
+  def wkbToScalaGeometry(wkb: Array[Byte]): Geometry = WKB.read(wkb)
```

Comment:
This isn't something that needs to be done now, but I'd be interested in seeing the performance difference between sending Geometries as `Array[Byte]` vs `String` between Scala/Python.